### PR TITLE
Rearrange slide panel to change tab order

### PR DESCRIPTION
**Pull request by amichaelparker**
### Summary
Addresses issue [#1272](https://github.com/cerner/terra-core/issues/1272) from [terra-core](https://github.com/cerner/terra-core). Divs are rearranged so that tabbing will focus main panel first, slide panel second.

Closes cerner/terra-core#1272

### CODE REVIEW

**Comment by bjankord** [comment]
@tbiethman Do you know if there is any concern with switching DOM order of panel and main content?

File: packages/terra-slide-panel/tests/jest/SlidePanel.test.jsx [hunk 1]
```diff
@@ -1,112 +0,0 @@
-import React from 'react';
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel/tests/jest/SlidePanel.test.jsx]: Do we want to remove these jest tests?

File: packages/terra-slide-panel-manager/tests/jest/SlidePanelManager.test.jsx [hunk 2]
```diff
@@ -1,68 +0,0 @@
-import React from 'react';
```
Comment (emilyrohrbough) [inline packages/terra-slide-panel-manager/tests/jest/SlidePanelManager.test.jsx]: Do we want to remove these jest tests?

**Comment by tbiethman** [comment]
The DOM position of the panel should be dependent upon the `panelPosition` value. I would assume that if the panel is in the `start` position, the existing DOM order would be preferred.

File: packages/terra-slide-panel/tests/jest/SlidePanel.test.jsx (hunk 1 above)
Comment (bjankord) [reply packages/terra-slide-panel/tests/jest/SlidePanel.test.jsx]: We should keep the jest tests @amichaelparker

File: packages/terra-slide-panel-manager/tests/jest/SlidePanelManager.test.jsx (hunk 2 above)
Comment (bjankord) [reply packages/terra-slide-panel-manager/tests/jest/SlidePanelManager.test.jsx]: We should keep the jest tests @amichaelparker

**Comment by mmalaker** [comment]
Functional verification completed. The slide panel examples now have the correct tab order.

**Pull request merged**